djmoney/__init__.py

```diff
@@ -1,2 +1,9 @@
-__version__ = "2.1.1"
-default_app_config = "djmoney.apps.MoneyConfig"
+import django
+
+if django.VERSION >= (3, 2):
+    # The declaration is only needed for older Django versions
+    pass
+else:
+    __version__ = "2.1.1"
```

Comment:
The `__version__` specification is for any version of Django. Will check up on other aspects. But this would have to be changed.

Reply:
Just following up on this.. it seems we don't import anything during setup.py so I would think that it would be okay to import django here